internal/services/automation/automation_variable_datetime_resource_test.go

```diff
@@ -77,6 +77,21 @@ func TestAccAutomationVariableDateTime_basicCompleteUpdate(t *testing.T) {
 	})
 }
 
+func TestAccAutomationVariableDateTime_encrypted(t *testing.T) {
```

Comment:
ditto with this comment https://github.com/hashicorp/terraform-provider-azurerm/pull/14887/files#r786637877